agent/utils/compare_versions.go

```diff
@@ -164,8 +164,8 @@ func compareSemver(lhs, rhs semver) int {
 	return 0
 }
 
-// extractVersion extracts a matching version from the version number string
-func extractVersion(input string) string {
+// ExtractVersion extracts a matching version from the version number string
```

Comment:
How come this file got moved?  I don't see `ExtractVersion` called anywhere but the existing places.